File: Tests.Common/CompilerGenerated/Member.cs
using Xunit;
using static ExpressionToString.Tests.Categories;

namespace ExpressionToString.Tests {
    public partial class CompilerGeneratedBase {
        [Fact]
        [Trait("Category", Member)]
        public void InstanceMember() => PreRunTest();

        [Fact]
        [Trait("Category", Member)]
        public void ClosedVariable() => PreRunTest();

        [Fact]
        [Trait("Category", Member)]
        public void StaticMember() => PreRunTest();

        [Fact(Skip ="Test for nested closure scopes")]
        [Trait("Category", Member)]
        public void NestedClosedVariable() {
            Assert.False(true);
        }
    }
}

File: Tests.Common.VB/CompilerGenerated/Member.vb
Partial Public Class VBCompilerGeneratedBase
    <Fact> <Trait("Category", Member)>
    Sub InstanceMember()
        PreRunTest()
    End Sub

    <Fact> <Trait("Category", Member)>
    Sub ClosedVariable()
        PreRunTest()
    End Sub

    <Fact> <Trait("Category", Member)>
    Sub StaticMember()
        PreRunTest()
    End Sub

    <Fact(Skip:="Test for nested scope")> <Trait("Category", Member)>
    Sub NestedClosureScope()
        Assert.False(True)
    End Sub
End Class
